.pre-commit-config.yaml

```diff
@@ -1,11 +1,19 @@
 minimum_pre_commit_version: "2.9.0"
 repos:
   - repo: "https://github.com/psf/black"
-    rev: "22.10.0"
+    rev: "21.12b0"
```

Comment:
Just to note that this is not enough to keep pre-commit to open a pull request every week, updating the version o black for us. 

One of the temporary workarounds (as far as I know) is to switch this to a `local` repo instead of the black GitHub URL. 

It would be something like this: 

```yaml
-   repo: local
    hooks:
    -   id: black
        name: black
        entry: black
        types: [python]
        language: system
        additional_dependencies: [click==7.1.2, black==21.12b0]
```

Although, I haven't tested this suggestion, but I think something like this would prevent pre-commit from auto-updating the version for black every week. See https://github.com/pre-commit/pre-commit-hooks/issues/332 for more details.

Reply:
@abadger

Reply:
One thing to note here. Newer black versions don't support running on v3.6. Meaning it won't work for my dev container PR 

#672

Reply:
You need the version `22.8.0` in #672, and you also downgraded the version for pre-commit-hooks... 

@abadger do you know if the issue that Andrew was having because of black is happening in `22.8.0`? If not, maybe we can try to pin that version down as it is more recent. 

As far #672, I would suggest you to do the same there, @SpyTec, at least for the pre-commit-hooks repo.

Reply:
21.12b0 is the latest version that won't break python2 code.  It looks like #672 is pinning a downgraded version of black so we should try to pin 21.12b0 there as well?

Reply:
Honestly, I think that for reducing complexity, we should create another PR that has the change and merge it right away... Since it's a change to the pre-commit config, we don't require any other CI check other than the pre-commit verification.